clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -531,6 +531,16 @@ protected void handleGroupCommand() {
 
     Command.setAuthenticationMode(auth, userName);
 
+    if (CommandActions.LIST.equals(command)) {
+      newListGroups(url, ignore, metalake).handle();
+      return;
+    }
+
+    if (Objects.isNull(group)) {
+      System.err.println("Group is undefined. Please use -g or --group to specify a group.");
```

Comment:
I would just say "Missing --group option." as that is in line with other error messages.

Reply:
fix, thank you, plz check again.